### PR TITLE
Work section dynamic project details

### PR DESCRIPTION
**Pull request by souad988**
Create project detail container 
Create data objects
Create functions to fill and append elements in project detail container dynamically
Add styling to this section

### CODE REVIEW

**Comment by elyor-doniyorov** [comment]
Hi @souad988, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -110,6 +110,28 @@ <h3 class="works_headline">My Recent Works</h3>
...
+      <!-- CARD WORK PROJECT DETAILS -->
+      <div id="work_project_details" class="work_project_details">
+        
+        <div class="work_project_details_card">
+          <span class="work_detail_close_icon">&times; </span>
+          <img class="work_project_details_img" src="./media/images/projects/youtube.PNG" alt="project screenshot"/>
+            <div class="work_project_details_title_btns"><h5 class="work_project_details_title"></h5>
+              <div class="work_project_details_btn_container btns_title">
+                <button type="button" class="buttons btns_title_live">See Live <img src="./media/Icons/seeliveicon.svg" alt="see live icon"/></button>
+                <button type="button" class="buttons btns_title_github">See Source <img src="./media/Icons/Icon-GitHub.svg" alt="see github icon"/></button>
```
Comment (ShoiraTa) [inline index.html:122]: - [ ] Please try to match the design of buttons with the one design provided in Figma: 

Figma  | Yours
------------- | -------------
image | image

File: index.html [hunk 2]
```diff
@@ -110,6 +110,28 @@ <h3 class="works_headline">My Recent Works</h3>
...
         />
       </div>
+      <!-- CARD WORK PROJECT DETAILS -->
+      <div id="work_project_details" class="work_project_details">
+        
+        <div class="work_project_details_card">
+          <span class="work_detail_close_icon">&times; </span>
+          <img class="work_project_details_img" src="./media/images/projects/youtube.PNG" alt="project screenshot"/>
+            <div class="work_project_details_title_btns"><h5 class="work_project_details_title"></h5>
+              <div class="work_project_details_btn_container btns_title">
```
Comment (ShoiraTa) [inline index.html:120]: - [ ] Please add the stack of technologies used for the project below the project tile:

Yours  | Figma
------------- | -------------
image| image

As you can see, yours is currently empty.

File: index.html [hunk 3]
```diff
@@ -110,6 +110,28 @@ <h3 class="works_headline">My Recent Works</h3>
...
+                <button type="button" class="buttons btns_title_live">See Live <img src="./media/Icons/seeliveicon.svg" alt="see live icon"/></button>
+                <button type="button" class="buttons btns_title_github">See Source <img src="./media/Icons/Icon-GitHub.svg" alt="see github icon"/></button>
+              </div>
+            </div>
```
Comment (ShoiraTa) [inline index.html:124]: - [ ] In figma you are provided with specifications on text design, you can find them on right side menu:

image

Please make sure to match the design of paragraph by giving the same specifications.

File: index.html (hunk 1 above)
Comment (ShoiraTa) [inline index.html:122]: - [ ] Please leave more space below buttons on mobile view: 

image

- [ ] Please check the margins and paddings of the content in Figma and try to apply them in your project 😉

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi team,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/project_details.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+{
+  LinkToSource: 'https://github.com/souad988/e-com',
+  linkToLiveVersion: 'https://souad988.github.io/e-com/',
+  image: './media/images/projects/youtube.png',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  name: 'ecom1',
+  technologies: ['Html', 'css', 'js', 'python', 'django'],
+}];
+
+export default projectsDetail;
```
Comment (rotimiazeez) [inline js/project_details.js:50]: - [ ] I'd suggest you follow the Figma design to improve the UI(user interface) of your page and kindly fix the broken image link too😎👍
- Your popup design 
screenshot (4) | screenshot (5)

- Figma design

screenshot (6) | screenshot (7)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM👋, 

### Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/project_details.js (hunk 4 above)
Comment (souad988) [reply js/project_details.js:50]: Hi @rotimiazeez 
thanks for reviewing, but for the imgs i don't know why they don't show up on github deployment while they do on my local machine if you can help with this i appreciate

**Review by Meltrust — APPROVED** [review]
Hi @souad988, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**